boards/x86/acrn/acrn.yaml

```diff
@@ -0,0 +1,6 @@
+identifier: acrn
+name: ACRN
+type: mcu
+arch: x86
+toolchain:
+  - zephyr
```

Comment:
should probably have flash/ram sizes here.

Reply:
done